README.md

```diff
@@ -0,0 +1,130 @@
+
+The sample supports:
+
+* Sending text, rich cards, and suggestions to consumers
+* Routing logic between bot (Dialogflow) and human answers
+* Customer satisfaction surveys
+* Administration console with live agent prototype
+* Quick setup of prerequisites and deployment to App Engine on behalf of the user
+
+This runs on the Google App Engine. See the [Google App Engine](https://cloud.google.com/appengine/docs/nodejs/) standard environment documentation for detailed instructions.
+
+## PREREQUISITES
+
+You must have the following software installed on your development machine:
+* [Google Cloud SDK](https://cloud.google.com/sdk) (aka gcloud)
+* [Node.js](https://nodejs.org/en/) - version 10 or above
+* [Python](https://www.python.org/) - version 3.7.0 or above
+* [Pip](https://pip.pypa.io/en/stable/installing/) (Python package manager)
+
+## SETUP
+
+### Register with Business Messages
+
+Open [Google Cloud Console](https://console.cloud.google.com) with your Business Messages Google account and create a new project for your agent. Note the **Project ID** and **Project number** values. Use https://***PROJECT ID***.appspot.com/callback as the webhook URL.
+
+1. [Register your project](https://developers.google.com/business-communications/business-messages/guides/set-up/register) with Business Messages. You may have to wait for a response from Google before continuing.
+2. Save the "Partner Key" provided by Google.
+
+### Setup billing
+
+1. Go to the [API Console](https://console.developers.google.com/) to view a list of existing projects.
+2. From the projects list, select a project or create a new one.
+3. Open the console left navigation menu and select **Billing**.
+4. Click **Enable billing** (if billing is already enabled then this option isn't available).
+5. If you don't have a billing account, create one.
+6. Select your location, fill out the form, and click **Submit and enable billing**.
+
+### Run project setup script
+
+This script handles enabling APIs, saving service accounts, enabling Firebase, and deploying to App Engine.
+
+1. Clone the Github repository:
+
+```bash
+$ git clone https://github.com/google-business-communications/bm-nodejs-dialogflow-integration
+```
+
+2. Run `gcloud init`, authorize your account, and choose the Google Cloud project created previously.
+3. Run `cd biab && pip install -r requirements.txt`.
+4. Run `python start.py <partner_key>`, where \<partner_key\> is the token saved earlier during the “Register with Business Messages” step.
+
+### Follow-up configuration
+
+Once the project setup script is finished and the app is deployed, it prints out
+
+    * Administration Console URL
+    * Firebase rule URL
+Save these URLs for the next step.
+
+If the administration console URL's domain is different from https://***PROJECT ID***.appspot.com/, then please contact Google to update your webhook URL.
```

Comment:
@thefalc please review this sentence, I added them.
We planned to use updatePartner in bot-in-a-box script, but I hesitated to use dev visibility client library.
So I added this explanation instead of adding it.

Reply:
Yeah, definitely don't want to use a feature that is not available yet. This seems fine.